command/command.go

```diff
@@ -28,16 +29,51 @@ import (
 
 var verbose = false
 
+type LogLevel string
+
+const (
+	INFO LogLevel = "INFO"
+	WARN LogLevel = "WARN"
+	ERROR LogLevel = "ERROR"
+)
+
 func printVerboseMessage(message string) {
 	if verbose {
 		fmt.Println(message)
 	}
+	logToFile(INFO, message)
 	logger := client.Config{}.Logger
 	if logger != nil {
 		logger.Debug(message)
 	}
 }
 
+/*
+This method Print the message to console and Logfile based on corresponding Log Level
+ */
```

Comment:
we're not using function describe comments any parts of our codebase fyi

Reply:
Removed from all places